src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -129,22 +129,22 @@ public partial interface ICompositionService
     public partial class ImportAttribute : System.Attribute
     {
         public ImportAttribute() { }
-        public ImportAttribute(string contractName) { }
-        public ImportAttribute(string contractName, System.Type contractType) { }
-        public ImportAttribute(System.Type contractType) { }
+        public ImportAttribute(string? contractName) { }
+        public ImportAttribute(string? contractName, System.Type? contractType) { }
```

Comment:
Same question on these

Reply:
This one also only unit test, this method doesn't have spceific doc, but above method documented to allow null
https://github.com/dotnet/runtime/blob/510113bc7f0a4a505ab0ccb90b0a593c3943baae/src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/ImportAttribute.cs#L73